src/utils.ts

```diff
@@ -26,8 +26,8 @@ export function normalizePath(id: string): string {
   return posix.normalize(slash(id))
 }
 
-export function resolve(...paths: string[]) {
-  return normalizePath(_resolve(...paths))
+export function resolve(root: string, path: string) {
```

Comment:
Honestly, I'm not sure if it's the right place, or if I should add one more helper function to do that. 
But this implementation work for my use case where I use ${configDir} in include, exclude, basePath, outDir.

Let me know if you think I should do it differently

Reply:
I think the general utility methods should not be modified. There should be a dedicated step to handle this feature.